src/mpid/ch4/src/ch4_impl.h

```diff
@@ -550,4 +550,12 @@ static inline uintptr_t MPIDI_CH4I_win_base_at_target(const MPIR_Win * win)
     return (uintptr_t) win->base;
 }
 
+static inline void MPIDI_find_tag_ep(MPIR_Comm* comm, int target_rank, int tag, int* ep_idx) {
+    *ep_idx = 0;
+}
+
+static inline void MPIDI_find_rma_ep(MPIR_Win* win, int target_rank, int* ep_idx) {
+    *ep_idx = 0;
+}
+
```

Comment:
Didn't we agree that find_ep function would take a parameter to specify a type of operation? Probably we can define it later though.
I think you tried to address at least a part of it by having two different functions for tag and rma.